index.php

```diff
@@ -273,6 +273,136 @@
                             </a>
                         </div>
                     </div> <!-- item end -->
+                    

```

Comment:
You should add a new section here. Check out this as reference https://github.com/KunSu/CMPE272MarketPlace/pull/20/files#diff-828e0013b8f3bc1bb22b4f57172b019dR290

Reply:
Done.